test/integration/targets/lineinfile/tasks/main.yml

```
@@ -253,6 +253,248 @@
     that:
       - "result.stat.checksum == 'ab56c210ea82839a54487464800fed4878cb2608'"
 
+###################################################################
```

This file is getting a bit long. Could you put these new tests in a separate tasks file?
```suggestion
- import_tasks: test_literal.yml
```

---

Moved to a separate file with specific block to keep both stages